validator/validator-main.protoascii

```diff
@@ -4524,7 +4504,7 @@ attr_lists: {
   # https://www.ampproject.org/docs/reference/components/amp-fx-collection
   attrs: {
     name: "amp-fx"
-    value_regex_casei: "(fade-in|parallax)"
+    value_casei: "parallax"
```

Comment:
Are we sure this is correct? I thought I saw recently one with two fade-in types.